charts/stable/shinobi/values.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: shinobisystems/shinobi
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: dev
+  # -- image pull policy
+  pullPolicy: Always
+
+# -- environment variables. See more environment variables in the [shinobi documentation](https://shinobi.video/docs/).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # PLUGIN_KEYS:
+  # SSL_ENABLED:
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        port: 8080
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
+# @default -- See values.yaml
+persistence:
+  config:
+    enabled: false
+    mountPath: /config
+  customautoload:
```

Comment:
```suggestion
  customAutoLoad:
```

Reply:
This fails some kind of validation. Maybe it should be `custom-auto-load`:
```
Invalid value: "customAutoLoad": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?'), spec.template.spec.containers[0].volumeMounts[1].name: Not found: "customAutoLoad"]
```